packages/manager/apps/key-management-service/src/components/layout-helpers/Create/OrderConfirmation.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import { useOrderURL } from '@ovh-ux/manager-module-order';
+import {
+  OsdsText,
+  OsdsButton,
+  OsdsLink,
+  OsdsIcon,
+  OsdsTile,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_ICON_SIZE,
+  ODS_ICON_NAME,
+  ODS_LINK_REFERRER_POLICY,
+} from '@ovhcloud/ods-components';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+} from '@ovhcloud/ods-common-theming';
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { ROUTES_URLS } from '@/routes/routes.constants';
+
+type OrderConfirmationProps = {
+  region: string;
+};
+
+const OrderConfirmation = ({ region }: OrderConfirmationProps) => {
+  const { t } = useTranslation('key-management-service/create');
+  const [orderLink, setOrderLink] = useState(null);
+  const orderBaseUrl = useOrderURL('express_review_base');
+  const navigate = useNavigate();
+  const generateExpressOrderLink = () => {
+    setOrderLink(
+      `${orderBaseUrl}?products=~(~(productId~'okms~planCode~'okms~duration~'P1M~pricingMode~'default~configuration~(~(label~'region~value~'${region}))))`,
```

Comment:
Maybe you can add it in the order lib directly
Like I did for the function `getVrackServicesProductSettings`